### ISSUE TITLE
Why is there both a README.md and an _init/README.md?

### ISSUE THREAD

**Issue by jeffgreco** [patronage/bubs#14]

**Comment by ccorda** [comment]
`README.md` is what you need to understand `bubs-wp`.  `_init/README.md` becomes the start of a specific site.

The former is what a dev evaluating Bubs might find useful.  The latter is meant for someone starting dev on a specific site.

Or that's the idea at least -- thoughts on how to improve?

**Comment by jeffgreco** [comment]
Yeah, it's a good plan overall to copy that in.  I think it would make sense to describe that in the initial README.md though, as I didn't think to check for a new README after the composer command.

**Comment by ccorda** [comment]
@jeffgreco passed along someone else struggled with this. reopening, we should be able to just have a single readme.

**Comment by jeffgreco** [comment]
Perhaps have one of the steps be "delete the earlier steps from this README"
